js/andrew/randomword.js

```diff
@@ -0,0 +1,287 @@
+var wordList = [
```

Comment:
Slightly inelegant to just have the dictionary here but otherwise very nicely done!